code/modules/jobs/job_types/adventurer/adventurer.dm

```diff
@@ -30,6 +28,11 @@ GLOBAL_VAR_INIT(adventurer_hugbox_duration_still, 3 MINUTES)
 	is_foreigner = TRUE
 	can_have_apprentices = FALSE
 
+/datum/job/adventurer/New()
```

Comment:
Do not make it random, I say leave it at 8 or 6, 8 for two classic adventurer parties since the migrant wave spawns 4 of them.